src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -1013,17 +1028,20 @@ private Map<String, BigDecimal> updateCandidateNamesInTally(RoundTally roundSumm
 
   private Map<String, BigDecimal> getInactiveJsonMap(RoundTally roundTally) {
     Map<String, BigDecimal> inactiveMap = new HashMap<>();
-    Pair<String, StatusForRound>[] statusesToPrint =
-        new Pair[] {
-          new Pair<>("overvotes",
-                     StatusForRound.INVALIDATED_BY_OVERVOTE),
-          new Pair<>("skippedRankings",
-                     StatusForRound.INVALIDATED_BY_SKIPPED_RANKING),
-          new Pair<>("repeatedRankings",
-                     StatusForRound.INVALIDATED_BY_REPEATED_RANKING),
-          new Pair<>("exhaustedChoices",
-                     StatusForRound.EXHAUSTED_CHOICE),
-        };
+    List<Pair<String, StatusForRound>> statusesToPrint = new ArrayList<>();
```

Comment:
There's something about this code (both the old and the new) that has me scratching my head. What would you think about moving these "key" strings into the `StatusForRound` enum as, say, `jsonKey`? Then the logic here might look like this:

```
private Map<String, BigDecimal> getInactiveJsonMap(RoundTally roundTally) {
    List<StatusForRound> statusesToPrint = new ArrayList<>();
    statusesToPrint.add(INVALIDATED_BY_OVERVOTE);
    statusesToPrint.add(INVALIDATED_BY_SKIPPED_RANKING);
    statusesToPrint.add(EXHAUSTED_CHOICE);
    statusesToPrint.add(INVALIDATED_BY_REPEATED_RANKING);

   if (config.usesSurpluses() && roundTally.getRoundNumber() == numRounds) {
      statusesToPrint.add(FINAL_ROUND_SURPLUS);
    }

    return statusesToPrint.stream()
        .collect(Collectors.toMap(StatusForRound::jsonKey, status -> roundTally.getBallotStatusTally(status)));
}
```

Reply:
Note that we have different strings for different outputs -- the JSON file uses camelCase and the CSV uses Title Plain Text (with spaces). I fear that this difference doesn't belong inside StatusForRound itself (e.g. `plaintextString` vs `camelCaseString`), though could see an argument for standardizing the text across the two formats. That would have a lot of downstream side-effects. Thoughts?

Reply:
Though #856 doesn't require different strings, I could see other future output formats possibly requiring other strings to match to each of the `StatusForRound` enums. The `.json` vs `.csv` formats are similar enough that it seems silly to have both, but other formats could be different enough and this spot seems as good a place as any.

Reply:
@artoonie You have more experience in this code than I do, so I'll certainly defer to your judgment. I'm just running off intuition here, and my intuition says that all of the different ways of titling these statuses should be held somewhere in common rather than spread around in different files. My concern is that by spreading these different strings around, we're likely to encourage a third or fourth formatting standard to crop up unnecessarily. But my familiarity with this code base is very thin, and I'm not here to block you!

Reply:
Got it! I tried it out and I do think it's cleaner as you suggested, though we previously had three different string types not two:
- camel case, brief (e.g. "overvote")
- title case, using "invalidated" (e.g. "Invalidated by Overvote")
- title case, using "inactive" (e.g. "Inactive by Overvote")

The "invalidated" phrase was only used in the logs. @yezr, I don't recall if that was an intentional choice to use it in the audit logs but not in the output CSV. Are you okay using "inactive" for both, or shall I add a third string to the enum in the last commit here?

Reply:
Using "inactive" for both will work.

Reply:
Great, then I'd like a final review on the last commit from Alyssa then I think we're ready to merge all three?

Thanks for the great feedback, Alyssa! This process has been super helpful.